rca/project_styleguide/templates/patterns/molecules/card/card.html

```diff
@@ -16,9 +16,18 @@
                     <div class="card__meta">{{ item.meta }}</div>
                 {% endif %}
                 <h3 class="card__heading">
-                    {% if item.link %}<span class="card__heading-link">{% endif %}
+                    {% if item.link %}
+                        <span class="card__heading-link">
+                            {{ item.title }}
+                            {% if link_style  == "with-arrow" %}
+                            <svg class="card__heading-link-icon" width="12" height="8">
+                                <use xlink:href="#arrow"></use>
+                            </svg>
+                            {% endif %}
+                        </span>
+                    {% else %}
                         {{ item.title }}
-                    {% if item.link %}</span>{% endif %}
+                    {% endif %}
```

Comment:
We could've just added just the SVG though it could've gotten a bit confusing. I've just updated this so that we just repeat `{{ item.title }}` instead.

It used to be:

```
{% if item.link %}<span class="card__heading-link">{% endif %}
    {{ item.title }}
{% if item.link %}</span>{% endif %}
```

Reply:
Makes sense 👍